README.md

```diff
@@ -78,6 +79,7 @@ Once the widget is rendering, additional configurations can be added to the Awes
 33. **`bestAPR`** - This config is optional. It defaults to `0` - <small>in percent (ex: `4.5` = 4.5% APR)</small>.
 34. **`widgetTemplateLT`** - This config is optional. It defaults to `or monthly payments as low as %%price%% with %%logo%% %%info%%`. Available templates include `%%price%%`, `%%logo%%`, `%%link%%`, `%%info%%`, `%%question-mark%%`, `%%line-break%%`.
 35. **`ltAltModalHTML`** - This config is optional. It defaults to our long-term lending modal.
+36. **`modalversion`** - This config is optional. It defaults to `color`. Alternative values include `grayscale`
```

Comment:
key should be `modalTheme` and it should actually appear higher - this second set of options starts with a condition to not use without discussing with Sezzle, but `modalTheme` is okay to change at merchant's discretion.